app/src/test/kotlin/fr/free/nrw/commons/login/LoginActivityUnitTests.kt

```diff
@@ -0,0 +1,98 @@
+package fr.free.nrw.commons.login
+
+import android.app.ProgressDialog
+import android.content.Context
+import android.view.MenuItem
+import fr.free.nrw.commons.R
+import fr.free.nrw.commons.TestCommonsApplication
+import fr.free.nrw.commons.auth.LoginActivity
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.annotation.Config
+import org.robolectric.fakes.RoboMenuItem
+import java.lang.reflect.Field
+
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [21], application = TestCommonsApplication::class)
+class LoginActivityUnitTests {
+
+    @Mock
+    private lateinit var activity: LoginActivity
+
+    @Mock
+    private lateinit var context: Context
```

Comment:
On line 46 we set the context to not a mock (I think?). Should this be annotated with `@Mock`?

Reply:
Thanks for pointing it out @domdomegg I forgot to remove the annotation.